### ISSUE TITLE
Steam IDE asks to install .NET 3.5

### ISSUE THREAD

**Issue by gnysek** [YoYoGames/GameMaker-Bugs#7511]
### Description

I've installed Steam version of IDE for first time, and I was surprised, that when opening it, it asked me to install .NET 3.5 (and mentioned that I have .NET 2.0 and .NET 3.0 only on my PC).

I thought that 2024.6+ IDE uses .NET 8.0 . Maybe 3.5 is put in some config file of Steam IDE? Seems like it's outdated.

This seems to be very low priority, as selecting "don't install" runs IDE without issues.

### Steps To Reproduce

1. Install GameMaker using Steam for first time
2. Try to run it
3. You will be asked if you want to install .NET 3.5 or risk continuing without it

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.8.0 (Monthly)

### Which operating system(s) are you seeing the problem on?

Windows 11

### Are you running GameMaker from inside your Steam library?

Yes

### Sample Package Attached?

- [X] I have attached my bug package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by YYDan** [comment]
Yes, I have just confirmed Steam is still set to ask for 3.5 and some other older versions of redistributables also. Will review those values.

Note that we will likely only change this for the current "GameMaker" product, in case anyone is still deliberately using any of the legacy "GMS2 Desktop"/"GMS2 Web"/"GMS2 Mobile"/"GMS2 UWP" installs (although they should not be, really...)

Thanks!

**Comment by YYDan** [comment]
Okay, so .NET 3.5 is still required due to the shader compiler tool. This requirement will hopefully be changed some time in the future, but I can't remove it for _this_ issue.

However, as part of _this_ issue:

1. .NET 4.5 will be unticked, as we're already shipping .NET 6 as part of the GM install
2. Change from requesting _VS2019 redist (inc 2017 and 2015)_ to _VS2022 redist (inc 2019, 2017, and 2015)_
3. Stop requesting VS2012 redist also

**Comment by gnysek** [comment]
As a side note - seems I don't have .NET 3.5 installed (since I was asked to do it), yet shaders are compiling for me for years, except this is only for some specific format (GLSL ES is my usual choice).
